touchforms/backend/xformserver.py

```diff
@@ -365,6 +367,5 @@ def main(port=DEFAULT_PORT, stale_window=DEFAULT_STALE_WINDOW, ext_mod=[], offli
 
     main(
         port=options.port,
-        stale_window=options.stale_window,
-        ext_mod=args
```

Comment:
only bhoma was using the command line extensions and bhoma has long since forked from being supported on master